schematic/store/synapse.py

```diff
@@ -572,16 +594,10 @@ def updateDatasetManifestFiles(self, sg: SchemaGenerator, datasetId: str, store:
         # note that if there is an existing manifest and there are files in the dataset
         # the columns Filename and entityId are assumed to be present in manifest schema
         # TODO: use idiomatic panda syntax
+
+        dataset_files, manifest = self.fill_in_entity_id_filename(datasetId, manifest)
         if dataset_files:
-            new_files = self._get_file_entityIds(dataset_files=dataset_files, only_new_files=True, manifest=manifest)
 
-            # update manifest so that it contain new files
-            new_files = pd.DataFrame(new_files)
-            manifest = (
-                pd.concat([manifest, new_files], sort=False)
-                .reset_index()
-                .drop("index", axis=1)
-            )
 
             # update the manifest file, so that it contains the relevant entity IDs
             if store:
```

Comment:
not sure why we would need `if store` logic here, but that should be a separate discussion